package.json

```diff
@@ -36,7 +36,7 @@
     "fs-extra": "^10.0.0",
     "jest": "^26.6.3",
     "json-schema": "^0.4.0",
-    "next": "^11.0.1",
+    "next": "^12.0.7",
```

Comment:
Let's get rid of resolutions

Reply:
Yes, if `next` is explicitly defined in the other packages then `resolutions` is redundant.

I'm happy to get this resolved ASAP, since we can always adjust later.